### PR TITLE
Add rz_log and rz_core_notify functions usable with not-formatted strings.

### PR DESCRIPTION
**Pull request by Rot127**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Adds functions which allow to use `rz_log()` and `rz_core_notify_...()` with simple strings (no `printf` like formatting).

**Test plan**

Added

**Closing issues**

...

### CODE REVIEW

File: librz/util/log.c [hunk 1]
```diff
@@ -272,3 +273,18 @@ RZ_API void rz_log(const char *funcname, const char *filename,
...
+ * \brief Logging function for plugins with no vargs support.
+ *
+ * \param funcname Contains the function name of the calling function
+ * \param filename Contains the filename that \p funcname is defined in
+ * \param lineno The line number that this log call is being made from in filename
+ * \param lvl Logging level for output
+ * \param tag A string placed at the beginning of the log message. If NULL, the log level is used.
+ * \param msg The log message.
+ */
+RZ_API void rz_log_bind(const char *funcname, const char *filename,
```
Comment (XVilka) [inline librz/util/log.c]: Why not just name it `rz_log_str()` instead?

**Review by XVilka — CHANGES_REQUESTED** [review]

File: librz/util/log.c (hunk 1 above)
Comment (wargio) [reply librz/util/log.c]: agreed. also this could be called instead of the actual fmt one.

Comment (Rot127) [reply librz/util/log.c]: Why simple, if one can make it confusing :D

File: librz/util/log.c [hunk 2]
```diff
@@ -272,3 +273,18 @@ RZ_API void rz_log(const char *funcname, const char *filename,
 	rz_vlog(funcname, filename, lineno, level, tag, fmtstr, args);
 	va_end(args);
 }
+
+/**
+ * \brief Logging function for plugins with no vargs support.
```
Comment (XVilka) [inline librz/util/log.c]: No need to mention plugins here at all. It just logs the string, that's it.

File: librz/core/core.c [hunk 3]
```diff
@@ -106,6 +106,18 @@ RZ_API void rz_core_notify_error(RZ_NONNULL RzCore *core, RZ_NONNULL const char
 	va_end(args);
 }
 
+RZ_API void rz_core_notify_begin_str(RZ_NONNULL RzCore *core, RZ_NONNULL const char *text) {
```
Comment (XVilka) [inline librz/core/core.c:115]: Doxygen for these? Just `\brief` should be enough.

**Review by XVilka — CHANGES_REQUESTED** [review]

**Comment by XVilka** [comment]
Rename PR title and description as well, please

**Review by XVilka — APPROVED** [review]

**Pull request merged**